pwndbg/commands/__init__.py

```diff
@@ -23,10 +23,26 @@
 
 commands = []
 
+def list_current_commands():
+    gdb.execute('set pagination off')
+    command_list = gdb.execute('help all', False, True).strip().split('\n')
+    existing_commands = set()
+    for line in command_list:
+        line = line.strip()
+        # Skip non-command entries
+        if len(line) == 0 or line.startswith('Command class:') or line.startswith('Unclassified commands'):
```

Comment:
While we don't care that much about it - this might fail on other GDB language versions.
If we don't have any better solution for that, it can stay as it is.

Reply:
The least bad way I've come up with is to take the 2nd line of output, split at ":" and use that to search for ignored lines and then do something based on empty lines for the last chunk. Or possibly just do it related to empty lines for every chunk.

Reply:
Lets keep it as it is for now.